apps/ssr-test-app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
>es5 [](http://example.com/codeflow?start=15&length=3)

Can this test app target a much newer ES version rather than the ancient ES5?

Reply:
2017 isn't that ancient is it? (Oh dear it was 6 years ago (☉_ ☉))